src/Microsoft.VisualStudio.AppDesigner/PropPages/PropertyControlData.vb

```diff
@@ -1537,9 +1538,24 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Debug.Assert(Extender IsNot Nothing)
 
             'Note: For property get we must pass in the extended object as the component.
-            Return Descriptor.GetValue(Extender)
+            Return GetValue(Descriptor, Extender)
```

Comment:
That sounds like a CPS bug - it should return consistent results regardless of whether property has been hidden. If we need to change code in these pages to handle the new project system - it's a sign that we've broken compat and will likely break other parts of the VS code base.